src/app.cpp

```diff
@@ -116,6 +116,37 @@ class CanFrame {
 
 };
 
+template<class ObjectHeaderGeneric>
+std::uint64_t calculate_ts_res(ObjectHeaderGeneric *oh)
+{
+	uint64_t ts_resol = 0;
+	switch (oh->objectFlags) {
+	case ObjectHeader::ObjectFlags::TimeTenMics:
+		ts_resol = 100000;
```

Comment:
Are you sure about the number of zeros here?

Reply:
Yes. This value defines the number of units per second. In the respective case TEN_MICS (10 ^ -5 s) the value must be 10 ^ 5 == 100000.